src/test/java/fi/otavanopisto/kuntaapi/test/server/integration/ptv/PhoneServiceChannelInTestsIT.java

```diff
@@ -0,0 +1,140 @@
+package fi.otavanopisto.kuntaapi.test.server.integration.ptv;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Collections;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+
+import com.github.tomakehurst.wiremock.junit.WireMockRule;
+import com.jayway.restassured.http.ContentType;
+
+import fi.metatavu.kuntaapi.server.rest.model.PhoneServiceChannel;
+import fi.metatavu.ptv.client.model.V7VmOpenApiPhoneChannel;
+import fi.metatavu.ptv.client.model.V7VmOpenApiPhoneChannelInBase;
+import fi.otavanopisto.kuntaapi.server.persistence.model.clients.AccessType;
+import fi.otavanopisto.kuntaapi.server.persistence.model.clients.ClientOrganizationPermission;
+import fi.otavanopisto.kuntaapi.test.AbstractPtvMocker;
+
+@SuppressWarnings ("squid:S1075")
+public class PhoneServiceChannelInTestsIT extends AbstractPtvInTest {
+
+  private static final String SERVICE_CHANNEL_FIND_PATH = "/phoneServiceChannels/{kuntaApiChannelId}";
+  /**
+   * Starts WireMock
+   */
+  @Rule
+  public WireMockRule wireMockRule = new WireMockRule(getWireMockPort());
+  
+  @Before
+  public void beforeTest() throws InterruptedException {
+    getPtvOrganizationMocker().mock(TestPtvConsts.ORGANIZATIONS);
+    getPtvServiceMocker().mock(TestPtvConsts.SERVICES);    
+    getPtvServiceChannelMocker().mock(TestPtvConsts.SERVICE_CHANNELS);
+
+    startMocks();
+    
+    waitApiListCount("/organizations", 3);
+    waitApiListCount("/phoneServiceChannels", TestPtvConsts.PHONE_SERVICE_CHANNELS.length);
+    waitApiListCount("/services", TestPtvConsts.SERVICES.length);
+  }
+  
+  @Test
+  public void updatePhoneServiceChannelUnauthorized() throws IOException, InterruptedException {
+    PhoneServiceChannel kuntaApiResource = getPhoneServiceChannel(0, TestPtvConsts.PHONE_SERVICE_CHANNELS.length);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)